Ciphers/AtBash.js

```diff
@@ -0,0 +1,26 @@
+const enAtbash = (message) => {
+  const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+  const tebahpla = 'ZYXWVUTSRQPONMLKJIHGFEDCBA'
+  const alphabet1 = 'abcdefghijklmnopqrstuvwxyz'
+  const tebahpla1 = 'zyxwvutsrqponmlkjihgfedcba'
+  let decodedString = ''
+
+  for (let i = 0; i < message.length; i++) {
+    const codedLetra = message.charAt(i)
```

Comment:
@aryabharat 
please add comments, hints, and proper filename.
thanks
@itsvinayak